src/Benchmarks/TimeOnlyToClockNotationConverterBenchmarks.cs

```diff
@@ -1,4 +1,4 @@
-﻿[MemoryDiagnoser(false)]
+[MemoryDiagnoser(false)]
```

Comment:
The first line of every file you modified seems to be changed. Is this a modified BOM? I'm not sure if there's a preferred version in the project, but you might want to check this.

Reply:
All I know is I didn't actually touch that part. 🤔 
Has VS some kind of default setting? If yes then that's what changed.
I don't see any problems happening here though and kinda don't know how I could change that. Git inside VS doesn't show them as changed. 
![grafik](https://github.com/user-attachments/assets/37e6b499-de0d-45de-a53c-dec452170536)